scripts/bootstrap

```diff
@@ -1,8 +1,8 @@
-#!/usr/bin/env bash
+#!/bin/sh
 
-set -e
+set -o errexit
 
-if $(docker volume inspect flipstone_stack_root >/dev/null 2>/dev/null); then
+if docker volume inspect flipstone_stack_root >/dev/null 2>/dev/null; then
```

Comment:
Interesting, so is this equivalent since we are in an `if`?

Reply:
They aren't strictly equivalent. The old version could evaluate the output it was more shell commands. `shellcheck` is pretty great for catching stuff like this.